packages/integrations/image/README.md

```diff
@@ -22,8 +22,22 @@ This integration provides `<Image />` and `<Picture>` components as well as a ba
 
 ## Installation
 
+Along with our integration, we recommend installing [sharp](https://sharp.pixelplumbing.com/) when appropriate. 
 
- ### Quick Install
+The `@astrojs/image` default image transformer is based on [Squoosh](https://github.com/GoogleChromeLabs/squoosh) and uses WebAssembly libraries to support most deployment environments, including those that do not support sharp, such as StackBlitz and Netlify Edge.
```

Comment:
I *think* Netlify Edge runs on deno so the integration won't work there, it still depends on Node even without sharp